R/derive_vars_aage.R

```diff
@@ -86,30 +86,47 @@ derive_vars_aage <- function(dataset,
 
 #' Derive Age in Years
 #'
-#' @details This function is used to convert age variables into years.
-#' These can then be used to create age groups.
+#' Converts the given age variable (`age_var`) to the unit 'years' from the current
+#' units given in the `{age_var}U` variable or `age_unit` argument and stores
```

Comment:
What is the `{age_var}U`?

Reply:
Sorry, I didn't think this was particularly clear when I wrote it but wasn't quite sure how else to word it...

Basically, if `age_var = AGE` then `{age_var}U = AGEU`. (I was following somewhat of the `glue` notation here but am happy to change if you have an idea as to what would be clearer?)

Reply:
ah I see.  Maybe something more explicit as some might not be familiar with that syntax.  Perhaps a `age_var+U`

Reply:
Makes sense - will update to that, thanks